### PR TITLE
doc: add document to explain how to run on GKE

### PR DESCRIPTION
**Pull request by tokers**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

___
### Backport patches
- Why need to backport?

- Source branch

- Related commits and pull requests

- Target branch

### CODE REVIEW

File: docs/deployments/gke.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+```shell
+cd /path/to/apisix-ingress-controller
+# install base resources, e.g. ServiceAccount.
+helm install ingress-apisix-base -n ingress-apisix ./charts/base
+# install apisix-ingress-controller
+helm install ingress-apisix ./charts/ingress-apisix \
+  --set ingressController.image.tag=dev \
+  --set ingressController.config.apisix.baseURL=http://apisix-admin:9180/apisix/admin \
+  --namespace ingress-apisix
+```
```
Comment (gxthrj) [inline docs/deployments/gke.md:71]: admin_key?

**Comment by tokers** [comment]
@moonming @membphis @gxthrj Please take a look when you have time.

File: docs/deployments/gke.md (hunk 1 above)
Comment (tokers) [reply docs/deployments/gke.md:72]: Added.

**Review by membphis — APPROVED** [review]

File: docs/deployments/gke.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+helm install ingress-apisix-base -n ingress-apisix ./charts/base
+# install apisix-ingress-controller
+helm install ingress-apisix ./charts/ingress-apisix \
+  --set ingressController.image.tag=dev \
+  --set ingressController.config.apisix.baseURL=http://apisix-admin:9180/apisix/admin \
+  --set ingressController.config.apisix.adminKey=edd1c9f034335f136f87ad84b625c8f1 \
+  --namespace ingress-apisix
+```
+
+Change the `ingressController.image.tag` to the Apache APISIX version that you desire. You have to Wait for while until the correspdoning pods are running.
```
Comment (membphis) [inline docs/deployments/gke.md:74]: `You have to Wait` -> `You have to wait`

Comment (tokers) [reply docs/deployments/gke.md:74]: OK.

Comment (tokers) [reply docs/deployments/gke.md]: Fixed.

**Review by gxthrj — APPROVED** [review]

**Pull request merged**